src/Annotation/AllowEmpty.php

```diff
@@ -13,28 +14,24 @@
  * \Laminas\InputFilter\Input should enable the allowEmpty flag.
  *
  * @Annotation
+ * @NamedArgumentConstructor
```

Comment:
Import is missing

Reply:
afaik, this annotation was marked deprecated and thus should be dropped anyways? 

afair using the NotEmpty validator was the way to go.

Reply:
See froschdesign's comment [here](https://github.com/laminas/laminas-form/pull/111#discussion_r632311433).

Reply:
Still the `use` statement for `@NamedArgumentConstructor` is missing from the other dozen Annotations, so please add it everywhere

Reply:
done